### ISSUE TITLE
Stack is thrown when restoring a session containing a norg file via rmagatti/auto-session

### ISSUE THREAD

**Issue by igorlfs** [nvim-neorg/neorg#567]
### Prerequisites

- [X] I have read [contribution guidelines](https://github.com/vhyrro/neorg/blob/main/docs/CONTRIBUTING.md)
- [X] I am using the latest version of the plugin
- [X] I am using either 0.6 stable or the latest compiled neovim version

### Neovim Version

NVIM v0.8.0-dev-994-g5ff2ea6687

### Neorg setup

```lua
require("neorg").setup({
    load = {
        ["core.highlights"] = {
            config = {
                todo_items_match_color = "all",
            },
        },
        ["core.defaults"] = {},
        ["core.norg.concealer"] = {},
    },
})
```

### Actual behavior

Opening neovim throws a stack trace:

```
Error executing vim.schedule lua callback: .../neorg/lua/neorg/modules/core/syntax/module.lua:190
: Invalid buffer id: 1                                                                                    
stack traceback:                                                                                          
        [C]: in function 'nvim_buf_get_name'                                                              
        .../neorg/lua/neorg/modules/core/syntax/module.lua:190: in function 'trigger_highlight_regex_code_block'                                                                                           
        .../neorg/lua/neorg/modules/core/syntax/module.lua:633: in function 'on_event'           
        .../neorg/lua/neorg/events.lua:153: in function <.../neorg/lua/neorg/events.lua:140>
```

### Expected behavior

No stack is thrown

### Steps to reproduce

1. Open a norg file
2. Save a session using rmagatti/auto-session (e.g., `:SaveSession`) and quit nvim
3. Reopen nvim, launching it without arguments (as to trigger auto-session)

### Potentially conflicting plugins

[rmagatti/auto-session](https://github.com/rmagatti/auto-session)

### Other information

Restoring a session after launching (e.g., `:RestoreSession`) does not trigger the stack.

### Help

No

**Comment by igorlfs** [comment]
By the way, this seems to be happening every time I launch nvim, given that I'm restoring a session (meaning I don't have to have a norg file loaded)

**Comment by igorlfs** [comment]
Fixed with d4e771d

**Comment by vhyrro** [comment]
Oh wow, I accidentally ended up fixing it :sweat_smile:. Glad it works now! Also sorry I missed your issue somehow, happens every now and again :(
